account_move_print/report/journal_report.xml

```diff
@@ -0,0 +1,176 @@
+                        <p class="m-0" t-field="o.partner_id.name" />
+                    </div>
+                </div>
+                <t t-set="total_debit" t-value="0.0" />
+                <t t-set="total_credit" t-value="0.0" />
+                <div class="row">
+                    <table name='lines_content' class="table table-sm o_main_table">
+                        <thead>
+                            <tr>
+                                <th name="th_account" class="text-left">Account</th>
+                                <th name="th_partner" class="text-left">Partner</th>
+                                <th name="th_label" class="text-left">Label</th>
+                                <th
+                                    name="th_amount_currency"
+                                    class="text-left"
+                                    groups="base.group_multi_currency"
+                                >Amount in Currency</th>
+                                <th name="th_debit" class="text-left">Debit</th>
+                                <th name="th_credit" class="text-left">Credit</th>
+                                <th
+                                    name="th_tax_id"
+                                    class="text-left"
+                                >Originator Tax</th>
+                                <th name="th_tax_ids" class="text-left">Taxes</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <t t-foreach="o.line_ids" t-as="line">
+                                <t
+                                    t-set="total_debit"
+                                    t-value="total_debit + line.debit"
+                                />
+                                <t
+                                    t-set="total_credit"
+                                    t-value="total_credit + line.credit"
+                                />
+                                <tr>
+                                    <td name='tb_account'>
+                                        <span t-field="line.account_id.display_name" />
+                                    </td>
+                                    <td name='tb_partner'>
+                                        <span t-field="line.partner_id.name" />
+                                    </td>
+                                    <td name='tb_line_name'>
+                                        <span t-field="line.name" />
+                                    </td>
+                                    <td
+                                        name='tb_amount_currency'
+                                        groups="base.group_multi_currency"
+                                    >
+                                        <t t-if="line.currency_id">
+                                            <span
+                                                t-esc="line.amount_currency"
```

Comment:
Replace this and all other `t-esc` by `t-out`, as the latter is kind of deprecated.